clang/lib/CIR/CodeGen/CIRGenStmt.cpp

```diff
@@ -551,20 +551,11 @@ mlir::LogicalResult CIRGenFunction::buildGotoStmt(const GotoStmt &S) {
   // info support just yet, look at this again once we have it.
   assert(builder.getInsertionBlock() && "not yet implemented");
 
-  mlir::Block *currBlock = builder.getBlock();
-  mlir::Block *gotoBlock = currBlock;
-  if (!currBlock->empty() &&
-      currBlock->back().hasTrait<mlir::OpTrait::IsTerminator>()) {
-    gotoBlock = builder.createBlock(builder.getBlock()->getParent());
-    builder.setInsertionPointToEnd(gotoBlock);
-  }
```

Comment:
This part of code was used for test `shouldCreateBlkForGoto` in `clang/test/CIR/CodeGen/goto.cpp`. However, it is unrelated to `goto`, but a workaround for the broken code for `break`. This patch also moves the test to `switch.cpp` and renamed it to `unreachable_after_break_1`.